codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/CodegenVisitor.kt

```diff
@@ -47,7 +53,6 @@ class CodegenVisitor(context: PluginContext) : ShapeVisitor.Default<Unit>() {
             .filter { integration -> integration.enabledForService(context.model, settings) }
             .onEach { integration -> LOGGER.info("Enabled KotlinIntegration: ${integration.javaClass.name}") }
             .sortedBy(KotlinIntegration::order)
-            .toList()
```

Comment:
question; why remove `toList()`?

Reply:
`sortedBy` returns a list so the `toList()` was unnecessary